src/app/app.module.ts

```diff
@@ -1,16 +1,30 @@
-import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
+import { RouterModule, Routes } from '@angular/router';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 import { AppComponent } from './app.component';
+import { MaterialModule } from './material/material.module';
+import { ActiveOrdersComponent } from './active-orders/active-orders.component';
```

Comment:
I think it would be better to create offers module `ng g module offers --routing`